src/Akka.Persistence.Sql.Hosting/HostingExtensions.cs

```diff
@@ -142,7 +145,8 @@ public static AkkaConfigurationBuilder WithSqlPersistence(
             DatabaseMapping? databaseMapping = null,
             TagMode? tagStorageMode = null,
             bool? deleteCompatibilityMode = null,
-            bool? useWriterUuidColumn = null)
+            bool? useWriterUuidColumn = null,
+            int? maxConcurrentQueries = null)
```

Comment:
Technically a breaking change without a method overload, but I don't think there's any other meta-packages built on top of Akka.Persistence.Sql.Hosting so it's probably fine as it's source-compatible